src/oas2/schema.ts

```diff
@@ -0,0 +1,22 @@
+import { isObject } from 'lodash';
+import { Schema } from 'swagger-schema-official';
+
+export function getExamplesFromSchema(data?: Schema) {
```

Comment:
Why is this possibility undefined? If you do not have a schema, there's no example, and the caller should know that.

```suggestion
export function getExamplesFromSchema(data: Schema) {
```

Reply:
Let's make the file name equal to the exported function.

Reply:
Removed undefined type and renamed file (also moved to transformers, though it's kind of util)